### ISSUE TITLE
models/research/struct2depth: pretrained model

### ISSUE THREAD

**Issue by HsuWanTing** [tensorflow/models#5888]
### System information
- **What is the top-level directory of the model you are using**: models/research/struct2depth
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04.5 LTS
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.11.0
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

### Describe the problem
1. need imagenet pretrained model for training, but don't no where can I download the imagenet checkpoint.
2. hope the author can release their pretrained depth model

**Comment by kwea123** [comment]
The pretrained depth models are [here](https://sites.google.com/view/struct2depth#h.p_agwk4jrb2a_M)

**Comment by sulashi** [comment]
I get the evaluation results of the two models using the evaluation code from [zhou](https://github.com/tinghuiz/SfMLearner) without the segmentation masks :
model-154688.npy
   abs_rel,     sq_rel,        rms,    log_rms,     d1_all,         a1,         a2,         a3
    0.1947,     1.4409,     6.3649,     0.2687,     0.0000,     0.7012,     0.9015,     0.9649
model-199160.npy
   abs_rel,     sq_rel,        rms,    log_rms,     d1_all,         a1,         a2,         a3
    0.1455,     1.1230,     5.3762,     0.2183,     0.0000,     0.8131,     0.9428,     0.9777

I think the two is not  the pretrained model on imagenet for training, they are the results of the paper. And I also hope the author can release their pretrained depth model.

**Comment by liyingliu** [comment]
@sulashi  Hi, may I know to get the following result, which portion of the KITTI dataset you have tested on by using this trained model model-199160?
abs_rel, sq_rel, rms, log_rms, d1_all, a1, a2, a3
0.1455, 1.1230, 5.3762, 0.2183, 0.0000, 0.8131, 0.9428, 0.9777

Is it the entire KITTI dataset or part of it?
Thanks.
